x-pack/legacy/plugins/triggers_actions_ui/public/index.scss

```diff
@@ -4,5 +4,6 @@
 // Styling within the app
 @import '../../../../plugins/triggers_actions_ui/public/application/sections/actions_connectors_list/components/index';
 
-@import '../../../../plugins/triggers_actions_ui/public/application/sections/action_connector_form/index';
+@import '../../../../plugins/triggers_actions_ui/public/application/sections/alert_add/index';
```

Comment:
FWIW Kibana now allows you to import scss directly into the components themselves. Rather than do this, you can actually just `import 'path/to/alert_add.scss` into the `expression.tsx` file (which should be near the typescript file now). This will mean it will only show on the files when loaded rather than entirely whenever the plugin as a whole loads.

I'll leave it to you if you wanna give it a shot. You would be able to do this all through the alerting app if you wanted to break it up a bit. Since alerting gets loaded in a lot of places, this would cut down the overall css load within Kibana.

If you need some more info on this, ping me on slack or something. I can walk you through it.

Reply:
I did that with all the .scss files listed on this file.